src/containers/CreateEventPage.js

```diff
@@ -125,6 +118,7 @@ export class CreateEventPage extends Component {
           name={name}
           category={category}
           eventFor={eventFor}
+          projects={this.props.projects}
```

Comment:
good catch, not sure how I didn't catch this before.
I noticed it this weekend, but since you have already put in the fix, it saves me time :smile_cat: